Spoony-iOS/Spoony-iOS/Source/Feature/Home/NMapView.swift

```diff
@@ -58,47 +75,95 @@ struct NMapView: UIViewRepresentable {
         return mapView
     }
     
+    private func configureMarkerCaption(_ marker: NMFMarker, with placeName: String, isSelected: Bool) {
+        marker.captionText = placeName
+        marker.captionColor = .black
+        marker.captionTextSize = 14
+        
+        if isSelected {
+            marker.captionMinZoom = 0
+        } else {
+            marker.captionMinZoom = 10
+        }
```

Comment:
삼항 연산자 사용해도 좋을 것 같아요! (개인적으로 삼항 연산자 선호)

```suggestion
marker.captionMinZoom = isSelected ? 0 : 10
```